<html>
  <head>
    <!-- Exception Handling -->
  </head>
  <body>
    <script>
      try {
        var a = 2;
        if (a == 2) document.write("ok");
        // a = 2 / 0;
        // console.log(a);
      } catch (Error) {
        document.write("Error found" + e.message);
      } 
      finally {
        document.write(" <br> Value of a is 2 ");
      }
        
    </script>
  </body>
</html>
